uk_election_timetables/election.py

```diff
@@ -61,27 +68,51 @@ def timetable(self) -> List[Dict]:
 
         :return: a list representing the entire timetable for this particular election.
         """
+        
         return sorted(
             [
                 {
                     "label": "Register to vote deadline",
                     "date": self.registration_deadline,
+                    "event": TimetableEvent.REGISTRATION_DEADLINE,
```

Comment:
Could you take the labels from the enum too? So we only have those strings once

Reply:
Meaning I should replace the label value with the enum value as in: 
`{
                    "date": self.registration_deadline,
                    "event": TimetableEvent.REGISTRATION_DEADLINE,
                    "label": TimetableEvent.REGISTRATION_DEADLINE.value,
                }`

Reply:
I always get confused with enums, but yeah, if that works? Then we're defining the type and label in one place